### PR TITLE
Forms/chioma/signin signup forms

### PR DESCRIPTION
**Pull request by Simpcyclassy**
This is my progress so far. I keep running into errors trying to remove the cancel and create buttons at the bottom.

### CODE REVIEW

File: src/components/signup/signup.js [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+//                     </Button>
+                
+//                 </div>
+          
+//             </Form>
+
+//         );
+
+//     }
+// }
```
Comment (Way2nnadi) [inline src/components/signup/signup.js]: delete unused code

File: src/components/signup/signup.js [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+
+import React, { Component } from 'react';
+import { Button, Modal, Form, Input, Tooltip, Icon, Checkbox } from 'antd';
+import { Title, Have_Account, Login, Read, Agreement } from './constant';
+import Signin from '../signin/Signin';
+import './style.css';
```
Comment (Way2nnadi) [inline src/components/signup/signup.js]: import `style.css` in the `components/index.js`. Reference coding conventions

File: src/components/signin/Signin.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import './style.css';
```
Comment (Way2nnadi) [inline src/components/signin/Signin.js]: import `style.css` in the `components/index.js`. Reference coding conventions

File: src/components/signin/Signin.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+  Checkbox
+} from 'antd';
+
+import {
+  Create_Account,
+  Title,
+  No_Account,
+  Reminder,
+  Forgot_Pass
+} from './constant';
```
Comment (Way2nnadi) [inline src/components/signin/Signin.js]: this import should be coming from `constants` not `constant`;

**Comment by EngrEric** [comment]
Why did you close the branch?

**Comment by Simpcyclassy** [comment]
@EngrEric, because I renamed

**Pull request closed without merging**